pickle2graphite.py

```diff
@@ -90,8 +90,8 @@ def __init__(self, metricsDict, base):
                 (year, month, day) = devAttrs["Date"].split("-")
                 (hour, minute, second) = devAttrs["Time"].split(":")
             except KeyError:
-                raise ValueError("Date or Time is missing or incorrrectly formatted for this set of metrics")
-                raise
```

Comment:
Absolutely agree re `tee` - the potential need to reprocess a pcap file, especially while in development / decipherment mode is something I learnt the hard way :-( 

Also re the seextract, although in my own personal setup I get by without using it, since 

* I listen passively to the solaredge traffic by means of port mirroring on a managed switch using a hardware setup which I adopted from the description here https://arachnoid.com/wireshark/ , and 
* I filter out the remaining unwanted traffic using tcp options as follows `sudo tcpdump -i $INTERFACE -U -X tcp and not port 22 -w -`